types/passport-local-mongoose/passport-local-mongoose-tests.ts

```diff
@@ -69,62 +68,62 @@ errorMessages.MissingUsernameError = 'No username was given';
 
-
 //#region Test Passport/Passport-Local
 passport.use(UserModel.createStrategy());
 
-passport.use('login', new LocalStrategy({
-        passReqToCallback: true,
-        usernameField: 'username',
-        passwordField: 'password'
-    },
-    (req: any, username: string, password: string, done: (err: any, res: any, msg?: any) => void) => {
-        process.nextTick(() => {
-            UserModel
-            .findOne({ 'username': username })
-            .exec((err: any, user: User) => {
-                if (err) {
-                    console.log(err);
-                    return done(err, null);
-                }
-
-                if (!user) {
-                    console.log(errorMessages.IncorrectUsernameError);
-                    return done(null, false, errorMessages.IncorrectUsernameError);
-                }
-
-                user.authenticate(password, function(autherr: any, authuser: User, autherrmsg: any) {
-                    if (autherr) {
-                        console.log(autherr);
-                        return done(autherr, null);
+passport.use(
+    'login',
+    new LocalStrategy(
+        {
+            passReqToCallback: true,
+            usernameField: 'username',
+            passwordField: 'password',
+        },
+        (req: any, username: string, password: string, done: (err: any, res: any, msg?: any) => void) => {
+            process.nextTick(() => {
+                UserModel.findOne({ username }).exec((err: any, user: User | null) => {
+                    if (err) {
+                        console.log(err);
+                        return done(err, null);
                     }
 
-                    if (!authuser) {
-                        console.log(errorMessages.IncorrectPasswordError);
-                        return done(null, false, errorMessages.IncorrectPasswordError);
+                    if (!user) {
+                        console.log(errorMessages.IncorrectUsernameError);
+                        return done(null, false, errorMessages.IncorrectUsernameError);
                     }
 
-                    return done(null, authuser);
+                    user.authenticate(password, function(autherr: any, authuser: User, autherrmsg: any) {
+                        if (autherr) {
+                            console.log(autherr);
+                            return done(autherr, null);
+                        }
+
+                        if (!authuser) {
+                            console.log(errorMessages.IncorrectPasswordError);
+                            return done(null, false, errorMessages.IncorrectPasswordError);
+                        }
+
+                        return done(null, authuser);
+                    });
                 });
             });
-        });
-    })
+        },
+    ),
 );
 
 type _User = User;
 
 declare global {
     namespace Express {
-        interface User extends PassportLocalModel<_User> {
-        }
+        // tslint:disable-next-line
```

Comment:
please add explicitely which rule is being disabled on next line

Reply:
Added each rule.